scripts/prolog

```diff
@@ -82,6 +82,26 @@ This graph depicts the relationship between the various `xxx-consensus-yyy` pack
 
 * __db-analyser__: A command-line tool for offline analysis of the consensus database
 
+=== Consensus implementation
+
+The comments in modules listed below provide important information on the
+implementation of consensus.
+
+* "Ouroboros.Consensus.Util.ResourceRegistry"
```

Comment:
Is double-quote enough to get a link from haddocks?

Reply:
It is :)